### PR TITLE
{bio}[foss/2018b] artic-ncov2019 v2020.04.13, rampart v1.2.0rc3, gotree v0.4.0, ... w/ Python 3.6.6

### PR DESCRIPTION
**Pull request by Darkless012**
(created using `eb --new-pr`)
requires: ~~#10458~~ (Clang) for Longshot, ~~#7735~~ (ETE), ~~#10469~~ (goalign, gotree), ~~#10464~~ (minimap2) for libdeflate

### CODE REVIEW

File: easybuild/easyconfigs/r/rampart/rampart-1.2.0rc3-foss-2018b-Python-3.6.6.eb [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('Porechop', '0.3.2pre', {
+        'source_tmpl': 'v%(version)s.tar.gz',
+        'source_urls': ['https://github.com/artic-network/Porechop/archive'],
+        'checksums': ['85980d6f37d38a44c66182e7b39bad487211ccfd8cb820c866ceed7ef7a15523'],
+    }),
+    ('binlorry', '1.3.1', {
+        'source_tmpl': '%(name)s-%(version)s-py3-none-any.whl',
```
[the remark this answers is not in the document]
Comment (Darkless012) [reply easybuild/easyconfigs/r/rampart/rampart-1.2.0rc3-foss-2018b-Python-3.6.6.eb:35]: Fixed, I've build it from source. Found only whl on pypi.

File: easybuild/easyconfigs/r/rampart/rampart-1.2.0rc3-foss-2018b-Python-3.6.6.eb [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        'unpack_sources': False,
+        'checksums': ['0aabdc2546a0dc7f298cc5c8b8c27e648a92c93f03d21ac424a7a845fc13ef5c'],
+    }),
+]
+
+# Don't ever try to install unpacked source of nodejs globally!
+# Always use 'npm install -g downloaded_file.tar.gz` or you will end up having broken links and installation will fail.
+# -d (debug) and --timing can show errors while downloading dependencies
+local_install_cmd = "npm install -d --timing --prefix %(installdir)s -g v%(version)s.tar.gz"
```
[the remark this answers is not in the document]
Comment (Darkless012) [reply easybuild/easyconfigs/r/rampart/rampart-1.2.0rc3-foss-2018b-Python-3.6.6.eb:44]: A lot, and I don't want to touch nodejs and its dependencies anymore.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 10 out of 10 (10 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, x86_64, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/c91731f172954dfbec0f9ec5043eade1 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 10 out of 10 (10 easyconfigs in this PR)
node3102.skitty.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/4f1b5fbebb7b9b4eda1368d577b8f407 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @Darkless012!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 15 out of 15 (10 easyconfigs in this PR)
node2500.golett.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz (haswell), Python 2.7.5
See https://gist.github.com/a653e5a4c6f8bf3b47bfd8b102be0824 for a full test report.

**Pull request merged**